mava/configs/logger/base_logger.yaml

```diff
@@ -9,3 +9,15 @@ use_neptune: True
 kwargs:
   neptune_project: Instadeep/Mava
   neptune_tag: [rware]
+
+# --- Checkpointing ---
+checkpointing:
+  save_model: True # Whether to save model checkpoints.
+  save_args:
+    save_interval_steps: 1 # Number of steps between saving checkpoints.
+    max_to_keep: 1 # Maximum number of checkpoints to keep.
+    keep_period: ~ # Don't delete any checkpoint where step % keep_period == 0
+
+  load_model: False # Whether to load model checkpoints.
+  load_args:
+    timestamp_override: "" # Timestamp override for loading checkpoint.
```

Comment:
This is not immediately apparent to me. Is possible to be a bit more descriptive with comment please?

Reply:
See the comment below on renaming this.

Reply:
Done :)